config.go

```diff
@@ -107,16 +60,26 @@ type Config struct {
 	// Optional. Default: "Bearer".
 	AuthScheme string
 
-	// KeyFunc defines a user-defined function that supplies the public key for a token validation.
+	// KeyFunc is a function that supplies the public key for JWT cryptographic verification.
 	// The function shall take care of verifying the signing algorithm and selecting the proper key.
-	// A user-defined KeyFunc can be useful if tokens are issued by an external party.
+	// Internally, github.com/MicahParks/keyfunc/v2 package is used project defaults. If you need more customization,
+	// you can provide a jwt.Keyfunc using that package or make your own implementation.
+	//
+	// This option is mutually exclusive with and takes precedence over JWKSetURLs, SigningKeys, and SigningKey.
+	KeyFunc jwt.Keyfunc // TODO Could be renamed to Keyfunc
```

Comment:
This exported field could be renamed to `Keyfunc`, which has the same capitalization as `jwt.Keyfunc`. A cosmetic change. I'm not sure if it's desired.

Reply:
I've removed the TODO.

Reply:
sounds good , use `Keyfun`